providers/src/airflow/providers/fab/auth_manager/cli_commands/utils.py

```diff
@@ -50,4 +54,12 @@ def get_application_builder() -> Generator[AirflowAppBuilder, None, None]:
     with flask_app.app_context():
         # Enable customizations in webserver_config.py to be applied via Flask.current_app.
         flask_app.config.from_pyfile(webserver_config, silent=True)
+        flask_app.config["SQLALCHEMY_DATABASE_URI"] = conf.get("database", "SQL_ALCHEMY_CONN")
+        url = make_url(flask_app.config["SQLALCHEMY_DATABASE_URI"])
+        if url.drivername == "sqlite" and url.database and not isabs(url.database):
+            raise AirflowConfigException(
+                f'Cannot use relative path: `{conf.get("database", "SQL_ALCHEMY_CONN")}` to connect to sqlite. '
```

Comment:
Will we be able to cover this using unit tests?

Reply:
Because it was a gap so far that wasn't caught, unit tests will ensure that we do not create that gap again

Reply:
sure

Reply:
Are you ok @amoghrajesh ? This BTW. Provider-only change - so we can release it independently from Airlfow .

Reply:
Yep, looks good now.

Reply:
> This BTW. Provider-only change - so we can release it independently from Airlfow .

Right, but its FAB. Might not be so simple. Better to involve @eladkal here

Reply:
:) Right, but its FAB. Might not be so simple.  :)